server/game/Card.js

```diff
@@ -584,6 +584,21 @@ class Card extends EffectSource {
         this.endRound();
     }
 
+    clearDependentCards() {
+        for (let upgrade of this.upgrades) {
+            upgrade.onLeavesPlay();
+            upgrade.owner.moveCard(upgrade, 'discard');
+        }
+
+        for (let child of this.childCards) {
+            child.onLeavesPlay();
+            child.owner.moveCard(child, 'discard');
+        }
+
+        this.purgedCards.forEach((c) => (c.purgedBy = null));
```

Comment:
much prefer if this for matched the ones above :)

(appreciate you'#re just copy/pasting but I've seen it now :D )

Reply:
Done.